### PR TITLE
Add "join table" for visits

### PR DESCRIPTION
**Pull request by nimplay**
Captura de pantalla 2022-09-13 a la(s) 8 23 32 p  m
Captura de pantalla 2022-09-13 a la(s) 8 12 57 p  m
Captura de pantalla 2022-09-13 a la(s) 8 15 49 p  m
Captura de pantalla 2022-09-13 a la(s) 8 24 27 p  m
Captura de pantalla 2022-09-13 a la(s) 6 38 49 p  m
Captura de pantalla 2022-09-13 a la(s) 6 42 14 p  m
Captura de pantalla 2022-09-13 a la(s) 8 16 33 p  m
Captura de pantalla 2022-09-13 a la(s) 8 25 04 p  m
Captura de pantalla 2022-09-13 a la(s) 7 50 20 p  m
Captura de pantalla 2022-09-13 a la(s) 7 51 24 p  m
Captura de pantalla 2022-09-13 a la(s) 8 17 43 p  m
Captura de pantalla 2022-09-13 a la(s) 8 26 01 p  m
Captura de pantalla 2022-09-13 a la(s) 8 03 19 p  m
Captura de pantalla 2022-09-13 a la(s) 8 21 21 p  m
Captura de pantalla 2022-09-13 a la(s) 8 26 44 p  m
Captura de pantalla 2022-09-13 a la(s) 8 05 31 p  m
Captura de pantalla 2022-09-13 a la(s) 8 22 05 p  m
Captura de pantalla 2022-09-13 a la(s) 8 27 18 p  m
Captura de pantalla 2022-09-13 a la(s) 8 07 09 p  m
Captura de pantalla 2022-09-13 a la(s) 8 22 43 p  m

- [ ] Create a table named vets
- [ ] Create a "join table" called specializations
- [ ] Create a "join table" called visits
- [ ] Insert data for vets
- [ ] Insert data for specialties
- [ ] Insert data for visits
 
Queries

- [ ] Who was the last animal seen by William Tatcher?
- [ ] How many different animals did Stephanie Mendez see?
- [ ] List all vets and their specialties, including vets with no specialties.
- [ ] List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
- [ ] What animal has the most visits to vets?
- [ ] Who was Maisy Smith's first visit?
- [ ] Details for most recent visit: animal information, vet information, and date of visit.
- [ ] How many visits were with a vet that did not specialize in that animal's species?
- [ ] What specialty should Maisy Smith consider getting? Look for the species she gets the most.

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,4 +1,20 @@
...
+    date_of_graduation DATE
+)
+
+CREATE TABLE visits (
+    visit_date DATE,
+    animals_id INT REFERENCES animals(id),
+    vets_id    INT REFERENCES vets(id)
+);
+
+
```
Comment (tresorsawasawa) [inline schema.sql]: - [ ] Kindly make sure you also add a new join table with the name **specializations** as required for **[the project](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_join_table.md)**.
Screenshot from 2022-09-14 02-44-30

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @nimplay 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow Gitflow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by nimplay** [comment]
Captura de pantalla 2022-09-13 a la(s) 8 59 29 p  m


specializations table have been created

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @nimplay 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Great job implementing the changes by the previous reviewer ✅

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**